pkg/controller/secrets/secrets.go

```diff
@@ -65,22 +65,22 @@ func GenerateUserData(privateKey []byte) (*core.Secret, error) {
 			$containerLogsPort = "10250"
 			New-NetFirewallRule -DisplayName $firewallRuleName -Direction Inbound -Action Allow -Protocol TCP -LocalPort $containerLogsPort -EdgeTraversalPolicy Allow
 			Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
-			Install-Module -Force OpenSSHUtils
-			Set-Service -Name ssh-agent -StartupType ‘Automatic’
 			Set-Service -Name sshd -StartupType ‘Automatic’
-			Start-Service ssh-agent
 			Start-Service sshd
 			$pubKeyConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PubkeyAuthentication yes','PubkeyAuthentication yes'
 			$pubKeyConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
- 			$passwordConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PasswordAuthentication yes','PasswordAuthentication yes'
+ 			$passwordConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PasswordAuthentication yes','PasswordAuthentication no'
```

Comment:
Whats going on here? Was this incorrect before?

Reply:
cant say that it was incorrect. But as a part of configuring ssh with pubkey, it would be preferable is we just use pubkey and disable password.

Reply:
Would argue that alignment with CoreOS would require also having passwd auth disabled by default:

```
core@voyager:~$ sudo grep PasswordAuthentication /etc/ssh/sshd_config | grep no
PasswordAuthentication no
core@voyager:~$ sudo grep VERSION /etc/os-release
VERSION="46.82.202010091720-0"
VERSION_ID="4.6"
OPENSHIFT_VERSION="4.6"
RHEL_VERSION="8.2"
REDHAT_BUGZILLA_PRODUCT_VERSION="4.6"
REDHAT_SUPPORT_PRODUCT_VERSION="4.6"
OSTREE_VERSION='46.82.202010091720-0'
```

Reply:
Agreed. However a config change should be made in a separate PR. We prefer not making too many changes in the same PR.